dlp/risk_test.py

```diff
@@ -12,18 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from gcp_devrel.testing.flaky import flaky
+from flaky import flaky
```

Comment:
@GoogleCloudPlatform/python-samples-owners I will bring this up in our meeting today but I also wanted to know more about this flaky package being used and why we don't use the regular one

Reply:
we determined it's cool to use the regular flaky project here! yay!